src/main/webapp/resources/js/components/pipeline/Pipeline.jsx

```diff
@@ -57,6 +57,10 @@ export default class Pipeline extends React.Component {
   };
 
   render() {
+    var pipelineUrl = `${window.TL.BASE_URL}pipelines/${this.props.id}`;
```

Comment:
This should not be in the render method as it only needs to be created the first time this component is built.  Move it up to the constructor.

Reply:
Done in 5954ce2